src/Raven.Server/Exceptions/IndexCreationException.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Raven.Server.Exceptions
+{
+    public class IndexCreationException: Exception
```

Comment:
Maybe it should be moved to Raven.Client.Exceptions.Documents.Indexes and inherit from RavenException so it would be propagated to client?

Reply:
Yes